### DISCUSSION TITLE
2nd CAN adapter with 3.3V and 5V

### DISCUSSION THREAD

**Discussion by herrfrei** [meatpiHQ/wican-fw#135] [General]
i, I would like to add a second CAN adapter to read a second bus. The device I would like to use is this one: https://soldered.com/product/can-transceiver-mcp2518-board/. 

It requires 5V as input voltage and includes a DC/DC converter to run the MCP2518 at 3.3V, so the signal levels should be fine. The question is where to get the 5V from. The internal connector gives us an number of IO pins that are usefule for my pupose. But where to get 5V? There is a pin labeled Vin - is that the raw voltage in the car (11..14V)?

![](https://user-images.githubusercontent.com/94690098/231444160-08842087-55ad-4165-8291-b379da63aeeb.png)

**Comment by meatpiHQ** [comment]
@herrfrei There is no 5V regulator on board, you will need a charge pump from 3v3 or a step down regulator from Vin which is the raw battery voltage.

**Reply by herrfrei** [reply]
Great, this is what I hoped. I have a 5V DC/DC converter, so I'll go on!
Thank you!
